src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/ProjectSystem/VS/Tree/Dependencies/Subscriptions/AnalyzerRuleHandler.cs

```diff
@@ -24,9 +24,9 @@ internal class AnalyzerRuleHandler : DependenciesRuleHandlerBase
             unresolvedIcon: ManagedImageMonikers.CodeInformationWarning,
             unresolvedExpandedIcon: ManagedImageMonikers.CodeInformationWarning);
 
-        protected override string UnresolvedRuleName { get; } = AnalyzerReference.SchemaName;
-        protected override string ResolvedRuleName { get; } = ResolvedAnalyzerReference.SchemaName;
-        public override string ProviderType { get; } = ProviderTypeString;
+        protected override string UnresolvedRuleName => AnalyzerReference.SchemaName;
+        protected override string ResolvedRuleName => ResolvedAnalyzerReference.SchemaName;
+        public override string ProviderType => ProviderTypeString;
```

Comment:
I assume the right-hand-sides here are all immutable?  Otherwise this is a change in behavior.

Reply:
Yes, they're all const strings.